staging/kos/pkg/apis/network/types.go

```diff
@@ -491,6 +491,9 @@ type IPLockSpec struct {
 	SubnetName string
 }
 
+// The only ExtendedObjectMeta section for an IPLock.
+const IPLockSectionWholeObj = "whole_object"
```

Comment:
I would make it just "spec" instead of "whole_object".  That way we do not have a problem if/when we introduce Status.